packages/synchro-charts/src/components/sc-dial/sc-dial-base/util.ts

```diff
@@ -0,0 +1,81 @@
+export enum StatusProgress {
```

Comment:
file name should be more descriptive - these are variables all relating to size, so instead maybe it should be called something like `sizeConfigurations` ?

Reply:
Done